app/assets/stylesheets/osem-schedule.css.scss

```diff
@@ -173,7 +173,7 @@ td.room{
 
 th.date{
   font-size: 12px;
-  padding: 8px 0px 8px 1% !important;
+  padding: 4px 0px 4px 2px !important;
```

Comment:
actually, it makes sense for the full-sized grids too, especially for the denser grids.
And in general, it seems to be more obvious to the end-user where the timestamp belongs in the cell, since the time in timestamp is where the cell starts (e.g its left border), when the timestamp is aligned to the center it looks more confusing IMO, please see our schedule with the left-aligned timestamps, we have exactly this style applied http://www.pgconf.us/conferences/2017/schedule